unstructured/partition/ocr.py

```diff
@@ -383,84 +437,66 @@ def zoom_image(image: PILImage, zoom: float = 1) -> PILImage:
     return PILImage.fromarray(new_image)
 
 
-def get_ocr_layout_from_image(
+def get_ocr_data_from_image(
```

Comment:
could you add docstring for this function? thanks

Reply:
Done!